app/views/communities/show.html.erb

```diff
@@ -1,6 +1,6 @@
-<div class="communities-show">
+<div id="<%= dom_id(@community) %>" class="communities-show">
 
-  <div class="jumbo light">
+  <div class="jumbo light" >
```

Comment:
Please, remove the blank space that is preceding the closing tag

Reply:
done